### PR TITLE
Flex charts

### PR DESCRIPTION
**Pull request by marliekeverweij**
Together with https://github.com/quintel/etsource/pull/2395

### CODE REVIEW

**Comment by antw** [comment]
I believe the Hound errors can be ignored. They mostly fall into two categories:

* Changes to files generated by Rails. I'm not going to fix violations in code style for these, since they'll likely get reset again the next time Rails updates them.
* The linter for older JS/CoffeeScript hasn't been changed to reflect our move to 100 character lines instead of 80.

There's also the issue that Hound doesn't have [eslint-plugin-unicorn](https://github.com/sindresorhus/eslint-plugin-unicorn). Since things seem to be moving kind of slowly in the Hound world, it might be time to move the code style checks to GitHub Actions.

File: config/locales/interface/output_elements/en_merit.yml [hunk 1]
```diff
@@ -123,6 +123,20 @@ en:
         demand.\r\n<br/><br/>\r\nThe table shows what place in the merit order each
         technology has, which is based on the operating costs. Also the table shows
         the available capacity of each technology as well as the full load hours."
+    flexibility_monthly_carriers_inflexible_supply_demand:
+      title: 1. Supply and demand per month for E, G, H<sub>2</sub> en H
```
Comment (antw) [inline config/locales/interface/output_elements/en_merit.yml:127]: Personally, I'm not a fan of the numbers being part of the chart titles; they won't mean anything to most visitors. To match the changes @michieldenhaan made to the Dutch translations

```suggestion
      title: Supply and demand per month
```

File: config/locales/interface/output_elements/en_merit.yml [hunk 2]
```diff
@@ -123,6 +123,20 @@ en:
...
+      description: ''
+    flexibility_monthly_carriers_inflexible_imbalance:
+      title: 2a. Imbalance per month for E, G, H<sub>2</sub> en H
```
Comment (antw) [inline config/locales/interface/output_elements/en_merit.yml:130]:
```suggestion
      title: Imbalance ("residual load") per month
```

File: config/locales/interface/output_elements/en_merit.yml [hunk 3]
```diff
@@ -123,6 +123,20 @@ en:
...
+      description: ''
+    flexibility_hourly_carriers_inflexible_imbalance:
+      title: 2b. Imbalance / residual load per hour for E, G, H<sub>2</sub> en H
```
Comment (antw) [inline config/locales/interface/output_elements/en_merit.yml:133]:
```suggestion
      title: Imbalance ("residual load") per hour
```

File: config/locales/interface/output_elements/en_merit.yml [hunk 4]
```diff
@@ -123,6 +123,20 @@ en:
...
+      description: ''
+    flexibility_hourly_all_carriers_cumulative_imbalance:
+      title: 3. Need for seasonal storage for E, G, H<sub>2</sub> en H
```
Comment (antw) [inline config/locales/interface/output_elements/en_merit.yml:136]:
```suggestion
      title: Need for seasonal storage
```

File: config/locales/interface/output_elements/en_merit.yml [hunk 5]
```diff
@@ -123,6 +123,20 @@ en:
...
+      description: 
+    flexibility_monthly_carriers_cumulative_imbalance:
+      title: 3. Cumulative imbalance per month for E, G, H<sub>2</sub> en H
```
Comment (antw) [inline config/locales/interface/output_elements/en_merit.yml:139]:
```suggestion
      title: Cumulative imbalance per month
```

File: config/locales/nl_node_details.yml [hunk 6]
```diff
@@ -0,0 +1,55 @@
+---
+nl:
```
Comment (antw) [inline config/locales/nl_node_details.yml:2]: Would anyone have time to add translations for these attribute names in the technical specs table?

Comment (michieldenhaan) [reply config/locales/nl_node_details.yml:2]: Done!

**Review by antw — APPROVED** [review]
Looks good to me. The data in the new charts is really great! Nice work @mabijkerk, @marliekeverweij, and @michieldenhaan!

:+1:

**Pull request merged**